sessionctx/variable/sysvar.go

```diff
@@ -1134,6 +1134,24 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(_ context.Context, s *SessionVars) (string, error) {
 		return strconv.FormatInt(MemoryUsageAlarmKeepRecordNum.Load(), 10), nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBPasswordReuseHistory, Value: strconv.Itoa(DefTiDBPasswordReuseHistory), Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxUint32, GetGlobal: func(_ context.Context, s *SessionVars) (string, error) {
```

Comment:
```suggestion
	{Scope: ScopeGlobal, Name: PasswordReuseHistory, Value: strconv.Itoa(DefPasswordReuseHistory), Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxUint32, GetGlobal: func(_ context.Context, s *SessionVars) (string, error) {
```

If the system variable is compatible with mysql, we don't need the prefix `TiDB`.

Reply:
done